src/Duende.Bff.EntityFramework/Duende.Bff.EntityFramework.csproj

```diff
@@ -40,11 +40,11 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="6.0.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="6.0.9" />
```

Comment:
@brockallen Please consider not doing updates like this. This changes the dependency of the package for, I assume, no reason. It's   an inconvenience to the consumers. 

Today I wanted to update my net6 application to 2.0.0-preview4 (from preview2), but nuget told me that I now need to update EFCore to 6.0.9. It means I now need to update AspNetCore to 6.0.9 as well as, for good measure, the SDK in the build pipeline. It's not a problem, but I feel it shouldn't be caused by a library update that is indifferent whether it gets 6.0.9 or 6.0.0.